### PR TITLE
add changelog lines for swarmkit and libnetwork

### PR DESCRIPTION
**Pull request by andrewhsu**
Add missing changelog lines for 1.12.2-rc3 release.

🐲

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ systemctl restart docker` to reload changes and (re)start the docker daemon.
  * Honor user provided listen address for gossip  [docker/libnetwork#1460](https://github.com/docker/libnetwork/pull/1460)
  * Allow reachability via published port across services on the same host [docker/libnetwork#1398](https://github.com/docker/libnetwork/pull/1398)
  * Change the ingress sandbox name from random id to just `ingress_sbox` [docker/libnetwork#1449](https://github.com/docker/libnetwork/pull/1449)
+ - Do not add names to ingress network [docker/libnetwork#1489](https://github.com/docker/libnetwork/pull/1489)
```
Comment (mlaventure) [inline CHANGELOG.md]: I'd specify service, e.g.: `Do not add service names...`

/cc  @mrjana

**Review by mlaventure — CHANGES_REQUESTED** [review]

**Comment by andrewhsu** [comment]
@mlaventure do we need verbiage for 3377b4b9b755082070c942cb2a73cec869f063ca ?

File: CHANGELOG.md (hunk 1 above)
Comment (andrewhsu) [inline CHANGELOG.md]: done

**Comment by mlaventure** [comment]
@andrewhsu it's already in the changelog: `Fix exec's children handling`

**Review by mlaventure — APPROVED** [review]

**Comment by mlaventure** [comment]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -56,6 +56,7 @@ systemctl restart docker` to reload changes and (re)start the docker daemon.
  * Honor user provided listen address for gossip  [docker/libnetwork#1460](https://github.com/docker/libnetwork/pull/1460)
  * Allow reachability via published port across services on the same host [docker/libnetwork#1398](https://github.com/docker/libnetwork/pull/1398)
  * Change the ingress sandbox name from random id to just `ingress_sbox` [docker/libnetwork#1449](https://github.com/docker/libnetwork/pull/1449)
+ - Do not add service names to ingress network [docker/libnetwork#1489](https://github.com/docker/libnetwork/pull/1489)
```
Comment (andrewhsu) [inline CHANGELOG.md]: done

**Comment by mavenugo** [comment]
@andrewhsu apologies on the delay. LGTM.

**Comment by mlaventure** [comment]
LGTM

**Comment by thaJeztah** [comment]
merged to master through https://github.com/docker/docker/pull/27328

**Pull request merged**